packages/custom-storage/src/sql/SQLStorage.ts

```diff
@@ -79,10 +79,8 @@ export class SQLStorage {
     return this._checkTable();
   }
 
-  public getVisualizations(sqlClient?: SQL): Promise<StoredVisualization[]> {
-    const client = sqlClient || this._sql;
-
-    return client.query(`
+  public getVisualizations(): Promise<StoredVisualization[]> {
```

Comment:
I do agree that it's better now, but I think that the parameter was meant for using a different SQL client with different credentials, and thus different permissions levels.

Reply:
I also thought about testing... who knows.

Managing a different sqlClient injected by param felt strange to me there. If we find the need we can re-add it in the future, but it would probably make sense just to create another SQLStorage instance